Configuration/StandardSequences/python/Accelerators_cff.py

```diff
@@ -4,3 +4,4 @@
 # used in production
 
 from HeterogeneousCore.CUDACore.ProcessAcceleratorCUDA_cfi import ProcessAcceleratorCUDA
+from HeterogeneousCore.ROCmCore.ProcessAcceleratorROCm_cfi import ProcessAcceleratorROCm
```

Comment:
Let me also ask a somewhat separate question: should we add
```py
from HeterogeneousCore.AlpakaCore.ProcessAcceleratorAlpaka_cfi import ProcessAcceleratorAlpaka
```
to this file already in this PR, or do it later?

Reply:
Ehm... what are the implications of doing it here, or later ?

Reply:
(with quick thought) Implications of doing now (when loading `Configuration.StandardSequences.Accelerators_cff`)
- we run a tiny bit more python code (from `ProcessAcceleratorAlpaka`)
- the `AlpakaService` service(s) are included in the process
- `ModuleTypeResolverAlpaka` code is run on the C++ side when modules are being constructed

without anything beyond test code actually using them (although I consider these costs tiny, given the near-future direction).

Implication of doing later
- in the mean time any configuration with Alpaka modules needs to explicitly load `HeterogeneousCore.AlpakaCore.ProcessAcceleratorAlpaka_cff`